src/graph/optimizer/OptimizerUtils.cpp

```diff
@@ -850,10 +850,30 @@ void OptimizerUtils::eraseInvalidIndexItems(
   }
 }
 
+bool OptimizerUtils::isBestIndex(const nebula::meta::cpp2::IndexItem& indexItem,
+                                 const std::vector<std::string>& returnCols) {
+  auto indexFields = indexItem.get_fields();
```

Comment:
`const auto&`

Reply:
In this scenario, using `player.name` or `player.age` has the same result that need to scan with data in the storaged. 
So I don't think there is much difference between the two choices though the number of lines scanned may be vary.